### DISCUSSION TITLE
Output via airplay?

### DISCUSSION THREAD

**Discussion by nic-nz** [vincentneo/LosslessSwitcher#119] [Q&A] [answered]
Apologies if this is an obvious question or has been covered elsewhere - will this enable sending highest res available per track / lossless via Airplay? I have read on Darko.audio that sending airplay 2 -> airplay 2 gets downsampled to lossy AAC 256kbps, so does this solve the problem therein? I'm assuming (but I don't know how to confirm?) my macs run airplay 2 (Mac mini 2014 Mojave 10.14 & MBP mid-2010 High Sierra). My setup means I need to send to a wiim amp, though I *could* direct connect the laptop. With great thanks for your efforts!

**Comment by MasterRahool** [comment, marked as answer]
Airplay is locked at 44.1k/16-bit so this app has no affect on it

The lossy aac compression thing i believe has to do with iOS or when using the airplay button in the Music app itself on Mac. Always enable airplay using the audio device list in the volume menu on mac.

**Reply by nic-nz** [reply]
Thank you for the detail and clarification- really appreciated.
